tools/oeutil/host/host.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#if defined(_WIN32)
+#define strcasecmp stricmp
+#endif
+#include "oeutil_u.h"
+
+#include "generate_evidence.h"
+
+FILE* log_file = nullptr;
+
+#define INPUT_PARAM_GENERATE_EVIDENCE "generate-evidence"
+#define SHORT_INPUT_PARAM_GENERATE_EVIDENCE_1 "generate"
+#define SHORT_INPUT_PARAM_GENERATE_EVIDENCE_2 "gen"
+
+typedef enum _oeutil_type
+{
+    OEUTIL_UNKNOWN = 0,
+    /**
+     * A debugging tool used to generate evidence, a report, or a certificate.
+     */
+    OEUTIL_GENERATE_EVIDENCE = 1,
+
+} oeutil_type_t;
+
+static void _display_help(const char* cmd)
+{
+    printf("Usage:\t%s <Util> <Options>\n", cmd);
+    printf("Util:\n");
+    printf(
+        "\t%s|%s|%s: generate evidence, a report, or a "
+        "certificate.\n",
+        INPUT_PARAM_GENERATE_EVIDENCE,
+        SHORT_INPUT_PARAM_GENERATE_EVIDENCE_1,
+        SHORT_INPUT_PARAM_GENERATE_EVIDENCE_2);
+    printf("Options:\n\tType oetuil <Util> --help for more information\n");
+}
+
+static oeutil_type_t _get_oeutil_type(int argc, const char* argv[])
+{
+    oeutil_type_t util_type = OEUTIL_UNKNOWN;
+
+    if (argc < 2)
+    {
+        printf("Invalid command.\n");
+        return util_type;
+    }
+
+    if (strcasecmp(INPUT_PARAM_GENERATE_EVIDENCE, argv[1]) == 0 ||
+        strcasecmp(SHORT_INPUT_PARAM_GENERATE_EVIDENCE_1, argv[1]) == 0 ||
+        strcasecmp(SHORT_INPUT_PARAM_GENERATE_EVIDENCE_2, argv[1]) == 0)
```

Comment:
replace this with strnicmp to do any substring match

Reply:
Thanks for the tip!